Fix incorrect scrolling when `Wrap` is set to `true`

dankox opened the pull request:
Fprint into the View doesn't scroll correctly content if `Autoscroll` and `Wrap` are turned on.

It's caused by using incorrect slice when checking length of lines if the content should be scrolled and how much.

There is new test for this called `wrap_autorscroll.go` which adds content to the view when hitting `Tab` key.

dankox committed:
Fix #92

Fprint into the View doesn't scroll correctly content if `Autoscroll` and
`Wrap` are turned on.
It's caused when checking buffer lines while using incorrect slice.

mjarkk commented on _examples/wrap_autoscroll.go:
```diff
@@ -0,0 +1,70 @@
+// Copyright 2014 The gocui Authors. All rights reserved.
```
Do we need this extra example?  
This seems more like a test for re-creating locally a problem.

dankox replied:
Yeah, you are right.... It kinda simulates the idea of autoscroll and wrap together.     

To be honest, you can recreate that by using `active.go` or `custom_frames.go` and makes your terminal small enough to force it to wrap the text in second view.    
So I can remove this exmple.

dankox committed:
Remove unnecessary example

This example just test the autorscroll and wrap together.
This can be recreated by using `active.go` or `custom_frames.go`
example and resize terminal so the text in the second view
will be forced to wrap.

jon4hz commented:
could you please make a new release with this fix?

mjarkk commented:
Ofc, here you go [v1.0.1](https://github.com/awesome-gocui/gocui/releases/tag/v1.0.1)
